apps/coscrad-frontend/src/components/Header/Header.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import { screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { getDummyConfigurableContent } from '../../app/App.spec';
+import { renderWithProviders } from '../../utils/test-utils';
+import { Header } from './Header';
+
+describe('Header', () => {
+    it('should load configurable content', () => {
+        const dummyConfigurableContent = getDummyConfigurableContent();
+        const siteTitle = 'The New Website Title';
+        const subTitle = 'The New Website Subtitle';
+        const myConfigurableContent = { ...dummyConfigurableContent, siteTitle, subTitle };
+
+        const { baseElement } = renderWithProviders(
+            <MemoryRouter>
+                <Header />
+            </MemoryRouter>,
+            {
+                contentConfig: myConfigurableContent,
+            }
+        );
+
+        const searchPattern = new RegExp(siteTitle);
```

Comment:
You need more spacing here.

Reply:
I'll add this but it will change in writing another test here.